supervision/keypoint/annotators.py

```diff
@@ -321,12 +323,19 @@ def annotate(
             skeletons_count=skeletons_count,
         )
 
+        text_colors = self.preprocess_and_validate_text_colors(
```

Comment:
@LinasKo, Should I call the `preprocess_and_validate_colors` method here too to get the list of `text_colors` then?